pyanaconda/core/util.py

```diff
@@ -85,7 +85,7 @@ def augmentEnv():
     return env
 
 
-def setSysroot(path):
+def set_system_root(path):
```

Comment:
Don't you want to rename this to `mount_system_root` or something similar. You are not setting path or anything now.

Reply:
I was thinking about it, but from the "user" point of view, you just want to tell Anaconda what is the actual path to the system root. The bind mount behind it is just an implementation detail, that could change again in the future. The name `mount_system_root` suggests that the real system root will be mounted somewhere and that is not correct.

Reply:
Still looks to me like a giant side effect. If we don't want to rename the method at least document that mount will happen and why it is happening.

Reply:
Fixed.